pkg/process/net/windows_pipe.go

```diff
@@ -17,10 +36,36 @@ type WindowsPipeListener struct {
 	pipePath string
 }
 
-// NewListener sets up a TCP listener for now, will eventually be a named pipe
-func NewListener(socketAddr string) (*WindowsPipeListener, error) {
-	l, err := net.Listen("tcp", socketAddr)
-	return &WindowsPipeListener{l, "path"}, err
+// systemProbePipeName is the effective named pipe path for system probe
+var systemProbePipeName = SystemProbePipeName
+
+// systemProbePipSecurityDescriptor has the effective DACL for the system probe named pipe.
+var systemProbePipSecurityDescriptor = namedPipeSecurityDescriptor
```

Comment:
Why do we need this assignment?

Reply:
To allow unit tests to override the named pipe and DACL.  Otherwise, the unit tests would hit the named pipe of the locally installed agent.